### ISSUE TITLE
Efficiently handle Relay's simplified-for-inference batch norms

### ISSUE THREAD

**Issue by gussmith23** [gussmith23/glenside#55]
Relay can run the `SimplifyInference` pass to simplify batch norm operators into a collection of primitive ops. Ideally, Glenside would intake these primitive ops and do efficient codegen for them. I don't think this is on the critical path for ISCA right now, and our workaround is to just take in opaque "BatchNormInference" nodes.
